app/tasks/maintenance_tasks/task.rb

```diff
@@ -72,6 +74,27 @@ def count
         new.count
       end
 
+      # Returns the list of the throttle specs for a given Task. This is
+      # provided as an array of hashes, with each hash specifying two keys:
+      # throttle_condition and backoff. Note that Tasks inherit throttle specs
+      # from their superclasses.
+
+      # @return [Array<Hash{Symbol => Proc, ActiveSupport::Duration}>]
+      def throttle_specs
+        @throttle_specs ||= superclass.throttle_specs.dup
```

Comment:
Could we use `class_attribute` instead?

Reply:
I always forget about `class_attribute`, thanks ❤️